### PR TITLE
[change] Chenged some files

### PR DESCRIPTION
**Pull request by urasakikeisuke**
## Pull Request Type

- change

## Description

Removed unnecessary task and added some ci badges

## Pre-Review Checklist for the Pull Request Author

- [x] Code follows coding-guidelines
- [x] Assign Pull Request to reviewer

## Checklist for the Pull Request Reviewer

- [x] Commits are properly organized and messages are according to the guideline
- [x] Code follows coding-guidelines
- [x] (Optional) Unit tests have been written for new behavior
- [ ] Pull Request title describes the changes

## Post-Review Checklist for the Pull Request Author

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write release notes

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ ]
+        language: [ 'python' ]
```
Comment (nakajima-john-shotaro) [inline .github/workflows/codeql-analysis.yml:35]: javascriptは入れなくても大丈夫？

**Review by nakajima-john-shotaro — APPROVED** [review]
Good work.🙃
By the way, do you want to talk about my job not getting finished?

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (urasakikeisuke) [reply .github/workflows/codeql-analysis.yml:35]: 了解しました。

**Comment by urasakikeisuke** [comment]
@nakajima-john-shotaro https://github.com/nakajima-john-shotaro/AIcon_dev/milestone/1

**Comment by urasakikeisuke** [comment]
@nakajima-john-shotaro A human being can live for 11 days without sleep.

**Pull request merged**